### ISSUE TITLE
Page.authenticate does not get all responses for Page.on('response') 

### ISSUE THREAD

**Issue by Edwardx89** [miyakogi/pyppeteer#139]
When using proxy authentication with page.authenticate the number of responses that I get back is significantly less than without proxy authentication.

```
async def main():
    browser = await launch()
    responses = []
    page = await browser.newPage()
    await page.authenticate({
                    'username': proxy_username,
                    'password': proxy_password
                })
    page.on('response', lambda res: responses.append(res))
    await page.goto('http://seamless.com')
    print(len(responses))
```

I couldn't figure out how or why certain responses are not being recorded when using page.authenticate vs not using proxies at all.

**Comment by miyakogi** [comment]
PR #148 may fix this.
Could you try with the latest dev branch?

```
python3 -m pip install -U git+https://github.com/miyakogi/pyppeteer@dev
```

**Comment by Edwardx89** [comment]
@miyakogi  I can confirm that PR#148 does fix this issue for me. Any timeline on when it'll be released?

**Comment by miyakogi** [comment]
If so, #142 might fix this issue.
Anyway, I have released version 0.0.25 including these patches, so close this issue.
